.pipelines/singletenancy/cilium-dualstack-overlay/cilium-dualstackoverlay-e2e-step-template.yaml

```diff
@@ -33,14 +33,16 @@ steps:
         kubectl cluster-info
         kubectl get po -owide -A
         echo "deploy Cilium ConfigMap"
-        kubectl apply -f test/integration/manifests/cilium/cilium-config-dualstack.yaml
-        echo "install Cilium ${CILIUM_VERSION_TAG}"
+        kubectl apply -f test/integration/manifests/cilium/v${CILIUM_TEMPLATE_DIR}/cilium-config/cilium-config-dualstack.yaml
+        echo "install Cilium ${CILIUM_DUALSTACK_VERSION}"
```

Comment:
`CILIUM_TEMPLATE_DIR` and `CILIUM_*_VERSION` match, right? If so, I think we should eliminate one of them to avoid having to maintain additional variables.

Reply:
these will have different values -- `CILIUM_TEMPLATE_DIR` is set to 1.14 and `CILIUM_*_VERSION` is 1.14.4, including the patch version